bin/ajax

```diff
@@ -858,9 +853,10 @@ if __name__ == '__main__':
     run_db_username = straxen.get_secret('RUNDB_USERNAME'.lower())
     run_db_password = straxen.get_secret('RUNDB_PASSWORD'.lower())
     run_client = pymongo.MongoClient(
-        f"mongodb://{run_db_username}:{run_db_password}@xenon1t-daq:27017,old-gw:27017/admin")
+        f"mongodb://{run_db_username}:{run_db_password}@xenon1t-daq:27017/xenonnt")

```

Comment:
I'm mildly confused that this worked in the past. Authenticating against the wrong database should fail.

Reply:
Sorry Darryl. This was a faulty commit that didn't work. I reverted it in d96c33c .

Reply:
Fair enough

Reply:
I was mixing the read-only account with the admin password. Of course we want to use the admin account if we need to update the database.